eva-accession-ws/src/main/java/uk/ac/ebi/eva/accession/ws/rest/ClusteredVariantsRestController.java

```diff
@@ -52,22 +55,37 @@ public ClusteredVariantsRestController(
         this.submittedVariantsService = submittedVariantsService;
     }
 
-    @ApiOperation(value = "Find clustered variants by identifier")
+    @ApiOperation(value = "Find clustered variants (RS) by identifier", notes = "This endpoint returns an accessioned" +
+            " clustered variant (RS). See " +
+            "https://github.com/EBIvariation/eva-accession/wiki/Import-accessions-from-dbSNP#clustered-variant-refsnp-or-rs " +
+            "for an explanation of each field.")
     @GetMapping(value = "/{identifiers}", produces = "application/json")
     public List<AccessionResponseDTO<ClusteredVariant, IClusteredVariant, String, Long>> get(
-            @PathVariable List<Long> identifiers) {
+            @PathVariable
+            @ApiParam(value = "List of numerical identifiers of clustered variants, e.g.: 3000000000,3000000002",
+                    required = true)
+                    List<Long> identifiers) {
         return basicRestController.get(identifiers);
     }
 
-    @ApiOperation(value = "Find submitted variants by clustered variant identifier")
+    @ApiOperation(value = "Find submitted variants (SS) by clustered variant identifier (RS)", notes = "This endpoint"
+            + " takes clustered variant identifiers (RS) and returns all accessioned submitted variants (SS) linked "
+            + "to the former. See "
+            + "https://github.com/EBIvariation/eva-accession/wiki/Import-accessions-from-dbSNP#submitted-variant-subsnp-or-ss "
+            + "for an explanation of each field.")
```

Comment:
Suggested rewording:

> Given a list of clustered variant identifiers (RS), this endpoint returns all the submitted variants (SS) linked to the former. For a description of the response, see https://github.com/EBIvariation/eva-accession/wiki/Import-accessions-from-dbSNP#submitted-variant-subsnp-or-ss

A similar paragraph could be used in the submitted-variants endpoint.